woocommerce/woocommerce-admin
# Deprecate the no longer used Gravatar component

I'm open to discussion about this component but we don't use it anymore so I think its pretty low value. As part of my goal to reduce the size of `@woocommerce/components` this seems like a good target for removal.


### Detailed test instructions:

1. Render a gravatar somewhere in the wc-admin application like the dashboard
2. navigate to the dashboard
3. Confirm that a deprecation warning is presented that tells the user Gravatar will be going away.


No changelog required (provided a changelog entry inside the package)
